soc/nordic/nrf53/Kconfig

```diff
@@ -246,10 +283,25 @@ config SOC_HFXO_CAP_DEFAULT
 
 config SOC_HFXO_CAP_EXTERNAL
 	bool "Use external load capacitors"
+	select DEPRECATED
+	help
+	  This option is deprecated, use DT instead. Example configuration:
+
+	  &lfxo {
+	      load-capacitors = "external";
+	  };
 
 config SOC_HFXO_CAP_INTERNAL
 	bool "Use internal load capacitors"
 	depends on NRF_SOC_SECURE_SUPPORTED
+	select DEPRECATED
+	help
+	  This option is deprecated, use DT instead. Example configuration:
+
+	  &lfxo {
+	      load-capacitors = "internal";
+	      ...
+	  };
```

Comment:
Maybe it would be better to refer to the SOC_HFXO_CAP_INT_VALUE_X2 option regarding the example configuration instead?

Reply:
added a comment